pageserver/src/tenant/timeline/walreceiver/connection_manager.rs

```diff
@@ -609,6 +609,17 @@ impl ConnectionManagerState {
     fn register_timeline_update(&mut self, timeline_update: SafekeeperTimelineInfo) {
         WALRECEIVER_BROKER_UPDATES.inc();
 
+        info!(
+            "Safekeeper info update: standby_horizon(cutoff)={}",
+            timeline_update.standby_horizon
+        );
+        if timeline_update.standby_horizon != 0 {
+            // ignore reports from safekeepers mnot connected to replicas
+            self.timeline
+                .standby_horizon
+                .store(Lsn(timeline_update.standby_horizon));
```

Comment:
This would be a great place to put a prometheus metric: I think currently we don't have any prometheus visibility for how far a standby is lagging, right?

Reply:
Added a metric.